extensions/vscode/src/views/homeView.ts

```diff
@@ -209,6 +209,9 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
         return await this.onVSCodeOpen(msg);
       case WebviewToHostMessageType.REQUEST_FILES_LISTS:
         return this.sendRefreshedFilesLists();
+      case WebviewToHostMessageType.REQUEST_CREDENTIALS:
+        await this.refreshCredentialData();
+        return this.updateWebViewViewCredentials();
```

Comment:
Can we follow the pattern moving these two lines into a function, to match the others we've maintained here?

Reply:
Updated in https://github.com/posit-dev/publisher/pull/2151/commits/d5ed8c9d15e59e0391e3c7d0d277ccd65989cfbf